src/main/java/playercreation/PlayerCreatorInputValidator.java

```diff
@@ -0,0 +1,75 @@
+package playercreation;
+
+import io.InputValidator;
+import objects.battle.SkillType;
+
+
+public class PlayerCreatorInputValidator implements InputValidator {
+    /** A class for validating input during Player creation at the start of the game.
+     * Attributes:
+     * currQuestion: The current question being asked of the user. An enum of type PlayerQuestion.
+     */
+    private final PlayerQuestion currQuestion;
+
+    public PlayerCreatorInputValidator(PlayerQuestion currQuestion) {
+        // Constructs a new PlayerCreatorInputValidator with currQuestion.
+        this.currQuestion = currQuestion;
+    }
+
+    @Override
+    public String parseAndValidate(String input) {
+        // Validates and parses user input if valid, returns null if invalid.
+        if (currQuestion == PlayerQuestion.SKILLTYPE) {
+            // Ensures the input corresponds to one of the SkillType enums.
+            if (input.equalsIgnoreCase(SkillType.AIR.name())) {
```

Comment:
its apparently recommended to use toString() instead since name() can't be overriden but it really doesnt matter